tools/azure-sdk-tools/devtools_testutils/proxy_startup.py

```diff
@@ -25,7 +25,7 @@
 CONTAINER_NAME = "ambitious_azsdk_test_proxy"
 LINUX_IMAGE_SOURCE_PREFIX = "azsdkengsys.azurecr.io/engsys/testproxy-lin"
 WINDOWS_IMAGE_SOURCE_PREFIX = "azsdkengsys.azurecr.io/engsys/testproxy-win"
-CONTAINER_STARTUP_TIMEOUT = 6000
+CONTAINER_STARTUP_TIMEOUT = 30
```

Comment:
Starting off being a bit more lenient; we'll see if we need to revisit this
```suggestion
CONTAINER_STARTUP_TIMEOUT = 60
```